src/Application.java

```diff
@@ -148,4 +156,45 @@ private void printResultSet(ResultSet res) throws SQLException{
             System.out.println();
         }
     }
+
```

Comment:
All the sql will have to be modified with the whole \" and ' thing to make it work. Other than that it looks good to me.